pkg/kvdpa/device.go

```diff
@@ -206,6 +208,16 @@ func GetVdpaDevicesByMgmtDev(busName, devName string) ([]VdpaDevice, error) {
 	return result, nil
 }
 
+/*GetVdpaDevicesByPciAddress returns the VdpaDevice objects for the given pciAddress */
+func GetVdpaDevicesByPciAddress(pciAddress string) ([]VdpaDevice, error) {
```

Comment:
The commit message clearly does not represent what's going on here. There are two new functions with clear functionality while the message seems to indicate it's a harmless refactoring.

Reply:
Rebasing again from master fixes the problem